Magento_Wishlist/templates/options_list.phtml

```diff
@@ -0,0 +1,80 @@
+<?php
+/* @var \Magento\Wishlist\Block\Customer\Wishlist\Item\Options $block */
+?>
+
+<?php $options = $block->getOptionList(); ?>
+<?php if ($options) : ?>
+
+<div
+    class="
+        more-info
+        wrapper
+        product-item-tooltip
+        margin-bottom-xs
+    "
+    data-mage-init='{"moreInfo": {}}'
+    aria-hidden="false"
```

Comment:
Not sure if it should be set to true

Reply:
I think it should be `false`. This `div` is always visible on the page. It should be accessible by screen readers because it contains `button` that opens more info section.

Reply:
ok great